matchingengine/OrderBook.go

```diff
@@ -135,7 +135,17 @@ func (ob *orderBook) AddAskOrder(ask *models.Ask) {
 	// in case of stoploss order, add it to stoploss queue and return
 	if ask.OrderType == models.StopLoss {
 		l.Debugf("Adding stopLoss with ask_id %d to the queue", ask.Id)
-		ob.askStoploss.Push(ask)
+		stockCopy, _ := models.GetStockCopy(ask.StockId)
+
+		if stockCopy.CurrentPrice <= ask.Price {
+
```

Comment:
remove blank line before block starting.

Reply:
use empty lines only to group lines of code. This doesn't help in grouping, so don't add here.